tech/languages/python/multiple-pythons.md

```diff
@@ -49,20 +50,19 @@ Note that most of the CPython versions, other than those provided by the
 `python2` and `python3` packages, are only provided for developer's convenience
 so you can run your test suites. Critical security bugs might not be always
 fixed and those interprets are not intended for production.
+Especially the development version might be quite unstable.
```

Comment:
Does it make sense to include it at all? I still wonder, if some beginning developer uses this page, is it useful to point him to unstable page?

Note, that I personally think that very few people read anything that's more than 2 lines/sentences long, so this may be very well overlooked.

Reply:
The point is to show you can develop (even as a beginning) against unstable 3.7 Python. However warning should be there. If they don't read them, well...

(That said, if removing this addition will make this PR more acceptable, tell me and I'll remove it.)

Reply:
Sure, sounds good to me. On the contrary, I'd push it to the top or even to standalone block, to make it more visible.

(Do not worry, it'll be merged alright. I'm just suggesting.)

Reply:
That actually is on topic :) Let me look at it.

Reply:
What about this:

**Warning:** The extra CPython versions (other than those provided by the `python2` and `python3` packages) are only provided for development, not production. They might be **unstable** or even **dangerous** (either because they are extremely old or quite the contrary fresh and alpha/beta quality).

Reply:
That's it, the 2 stable versions are `python2` and `python3`. The extra versions (i.e. the other than those provided by those packages) are considered "unstable" (development only).

Reply:
Sorry, I missed this somehow first time. I like the warning version more, how about reformulating it a bit like:

**Warning:** For production purposes you should use `python2` and `python3` packages. Other CPython versions might be **unstable** or even **dangerous** (either because they are extremely old or quite the contrary alpha/beta quality).

Reply:
Yes! I'd switch python3 to be first and maybe add "and are intended for development purposes only" to the end.

Reply:
Or "and are intended solely for development"

Reply:
c9079d1